### ISSUE TITLE
Style/YodaCondition with safe navigation operator bug

### ISSUE THREAD

**Issue by AlexWayfer** [bbatsov/rubocop#4411]
## Expected behavior

No warnings.

## Actual behavior

> Reverse the order of the operands `b&.value != 2`. (Style/YodaCondition)

## Steps to reproduce the problem

```ruby
a = 1 if b == 2 # it's ok
a = 1 if b.value == 2 # it's ok
a = 1 if b&.value == 2 # Warning
```

## RuboCop version

```
$ rubocop -V
0.49.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-linux)
```
